### ISSUE TITLE
Change Center frequency, from webpage

### ISSUE THREAD

**Issue by lu9cbl** [simonyiszk/openwebrx#102]
I'm cant change the center frequency on the web page, if i want to, i need to change the config file in the console, and then restart the process.

Thanks!

**Comment by ha7ilm** [comment]
That's on my TODO list, but I'm making progress slowly.

Cheers,

Andras, HA7ILM

**Comment by lu9cbl** [comment]
If i can help you, let me know. Thanks

**Comment by Giorgiofox** [comment]
Many peoples waiting this ;) I think this is a very important feature, please.

Thanks in advance
Giorgio

**Comment by Gymnae** [comment]
I'd also would immediately deploy this for my HF+ instead of using SpyServer if frequency and some of the settings KiwiSDR offers are available in this package.

It's a real pitty that @jks-prv did not commit to this project after the fork.

**Comment by Giorgiofox** [comment]
Any update for this?

**Comment by ha7ilm** [comment]
No update yet.

**Comment by Gymnae** [comment]
I see that there are some forks and wrappers trying to add this feature, but non really work. 
Has anyone found a way to allow frequency change in the frontend? This feature would be really great.

My AirSpyHF+ is so Windows centric, that using it via Network on MacOS or Linux is no fun - openwebrx is the better way to make use of this hardware.

**Comment by npaisnel** [comment]
Yes, just to add my voice to this request
The ability to change centre frequency from the webgui without having to ssh in and edit the config file would make this perfect for me.

I just want to monitor three separate frequencies  -all AM.  119.45mhz, 125.200 and 134.675
I keep my sample rate down to 250khz, running on a Pi 

To have these frequencies in a drop down bookmark list would be great too

**Comment by KC9S** [comment]
I agree.  This is excellent software.  Really looking forward to center frequency tuning.  That’s really the only feature that I desire.  Let me know if I can help!

**Comment by tarator** [comment]
+1 for this feature

**Comment by Gymnae** [comment]
I've added it to my fork, based on the work of yet another fork :) It works nicely with Airspy and sdrplay devices in addition to the classis rtl-sdr:
https://github.com/gymnae/openwebrx
